### ISSUE TITLE
pfe-codeblock | update theme colors for accessibility

### ISSUE THREAD

**Issue by alistairmc** [patternfly/patternfly-elements#1410]
## Description of the issue

Some of the colours in the code block do not meet the accessibility standard please update so colour contrast meets the standard.

### Impacted component(s)

- pfe-codeblock

### Steps to reproduce

Run any accessibility tool against the component demo page.

### Expected behavior

Colour contrast tests pass

Colours should be as follows:

### Light theme

$custom-pink: #B300B3; // `h1, p, body`
$custom-red: #C9190B; // `"javascript", red, #eee`
~~$custom-orange: #EC861B;~~ // `Use $custom-pink`
$custom-blue: #06C; // `<, </, >, height=""`
~~$custom-teal: #39ADB5;~~ // `Use $custom-blue`
$custom-purple: #40199A; // `const, let, null, main, printf, return`
$custom-white: #F5F5F5; // Background
$custom-gray: #6A6E73; // `example, text, #include, stdio.h, int`
$custom-light-gray: #6A6E73; // Numbers and dividers
$custom-black: #151515; // `example html, some paragraph text`
Font: Consolas / 16pt / 24 (1.5)

https://xd.adobe.com/view/166cbba5-d3c9-4068-938e-6941c26c39c9-433f/specs/

### Dark theme

$custom-red: #FF8A80; // `h1, p, body`
~~$custom-orange: #FE9A8C; //(red, #eee, etc.)~~ // `Use $custom-green`
~~$custom-yellow: #FFCB6B; //(height, width, cx, cy, etc.)~~ // `Use $custom-blue`
$custom-green: #ACE12E; // `"javascript", red, #eee`
$custom-blue: #73BCF7; // `<, </, >, height=""`
~~$custom-teal: #80CBC4;~~ // `Use $custom-blue`
$custom-purple: #B2A3FF; // `const, let, null, main, printf, return`
$custom-black: #3C3F42; // Background
$custom-gray: #D2D2D2; // `example, text, #include, stdio.h, int`
$custom-dark-gray: #D2D2D2; // Numbers and dividers
$custom-white: #FFF; // `example html, some paragraph text`
Font: Consolas / 16pt / 24 (1.5)

$custom-hilight-bg: #363636; // Not sure where this maps

https://xd.adobe.com/view/166cbba5-d3c9-4068-938e-6941c26c39c9-433f/screen/8117f58c-c2ad-4156-b52f-d0c6e21cc27a/specs/

**Comment by coreyvickery** [comment]
@alistairmc Updated your issue and added links to XD specs.

**Comment by alistairmc** [comment]
@coreyvickery thanks for the update, one question for the colours you have deleted orange, teal, and yellow. what colour do they map to?

**Comment by coreyvickery** [comment]
@alistairmc Do you happen to still have a link to my old XD design so I can compare?

**Comment by coreyvickery** [comment]
@alistairmc Made final updates, you should be all set.

**Comment by alistairmc** [comment]
@coreyvickery regarding the font I am inheriting the PFE   font-family--code:    '"Overpass Mono", Consolas, Monaco, "Andale Mono", monospace', is this not correct?

**Comment by alistairmc** [comment]
@coreyvickery why do you want the fonts set with px values, they are set to 1em and 1.5em and this is the same as 16 and 24px?

**Comment by coreyvickery** [comment]
@alistairmc Overpass is fine for now and 1em (16px) is fine for all sizes too.
